base_layer/core/tests/helpers/block_builders.rs

```diff
@@ -128,8 +133,12 @@ pub fn create_act_gen_block() {
 ///
 /// Right now this function does not use consensus rules to generate the block. The coinbase output has an arbitrary
 /// value, and the maturity is zero.
-pub fn create_genesis_block(factories: &CryptoFactories) -> (Block, UnblindedOutput) {
-    create_genesis_block_with_coinbase_value(factories, 100_000_000.into(), 100)
+pub fn create_genesis_block(
+    factories: &CryptoFactories,
+    consensus_constants: &ConsensusConstants,
+) -> (Block, UnblindedOutput)
+{
+    create_genesis_block_with_coinbase_value(factories, 100_000_000.into(), consensus_constants)
```

Comment:
In theory we could use the emission curve now

Reply:
that would force the unit tests to create custom curves as well. Do we want to go down that route?

Reply:
You're already passing `consensus_constants` in, so the emission is technically defined already. Tests should have to do anything unless they're using the 100e6 value already

Reply:
done